backend-server/src/api/data/dataAccess.ts

```diff
@@ -56,10 +62,54 @@ class DataAccess {
     );
     await setDoc(doc, interviewer);
   }
+
+  async eventDocRef( 
+    organization: string,
+    eventUID: string
+  ): Promise<DocumentReference<DocumentData>> {
+    return await doc(
+      this.rootCollection,
+      organization,
+      this.eventCollectionName,
+      eventUID
+    );
+  }
+
+  async getEvent(
+    organization: string,
+    eventUID: string
+    ): Promise<DocumentData> {
+      const doc = await this.eventDocRef(organization, eventUID);
+      const res = await getDoc(doc);
+      return res.data();
+    }
+
+  async setEvent(event: Event) {
+    const doc = await this.eventDocRef(
+      event.organization,
+      event.eventUID
+    );
+
+    await setDoc(doc, event);
+  }
+  async bookEvent(
+    organization: string,
+    eventUID: string,
+    requestedTime: string
+  ) {
+    const doc = await this.eventDocRef(organization, eventUID);
+    const confirmedTimeStamp = Timestamp.fromDate(new Date(requestedTime));
+    await setDoc(doc, 
+      { confirmedTime: confirmedTimeStamp },
```

Comment:
What is `confirmedTime` and `merge`? For booking, we will need to use Jin's function I think, so let's wait on that before doing this part